main/datalog.go

```diff
@@ -216,16 +216,17 @@ func bulkInsert(tbl string, db *sql.DB) (res sql.Result, err error) {
 
 	batchVals := insertBatchIfs[tbl]
 	for len(batchVals) > 0 {
-		i := int(0) // Maximum of 25 rows per INSERT statement.
+		i := int(0)
 		stmt := ""
 		vals := make([]interface{}, 0)
-		querySize := uint64(0) // Size of the query in bytes.
-		for len(batchVals) > 0 && i < 1000 && querySize < 750000 {
+		querySize := uint64(0)                                   // Size of the query in bytes.
+		for len(batchVals) > 0 && i < 30 && querySize < 750000 { // Maximum of 1000? columns for INSERT on default sqlite compile. trafficData is limiting function with 31 parameters.
```

Comment:
Catching up comments to Slack convo - there seems to be quite a bit of diminishing returns in batching a lot of rows, so increasing it to batch insert all rows at once didn't seem to make overall speed faster. At least in my tests, may not be true for all systems. There was *some* benefit to batching. Added code to master now that calculates the max rows based on the `999` column limitation and the number of columns being inserted into the current table.

OK, we might want to roll back some of the logging. Maybe we should implement verbosity levels?